README.md

```diff
@@ -264,7 +264,7 @@ See [Getting Support](https://success.jitterbit.com/display/DOC/Getting+Support)
 
 ## Copyright
 
-Copyright © 2018-2020 Jitterbit, Inc.
+Copyright © 2018-2019 Jitterbit, Inc.
```

Comment:
This one should be

    Copyright © 2018-2020 Jitterbit, Inc.

Reply:
Added copyright to all edited files.